From 1bed7fd04e44943511e50d82c59ca182d2532af1 Mon Sep 17 00:00:00 2001
From: Sander Vocke <sandervocke@gmail.com>
Date: Tue, 5 Sep 2023 06:32:01 +0000
Subject: [PATCH] Fix getpid.c path in Makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a424ef7..d9ef327 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ LIBWASI_EMULATED_MMAN_SOURCES = \
 LIBWASI_EMULATED_PROCESS_CLOCKS_SOURCES = \
     $(sort $(shell find $(LIBC_BOTTOM_HALF_DIR)/clocks -name \*.c))
 LIBWASI_EMULATED_GETPID_SOURCES = \
-    $(sort $(shell find $(LIBC_BOTTOM_HALF_DIR)/getpid -name \*.c))
+    $(sort $(shell find $(LIBC_BOTTOM_HALF_DIR)/sources/getpid -name \*.c))
 LIBWASI_EMULATED_SIGNAL_SOURCES = \
     $(sort $(shell find $(LIBC_BOTTOM_HALF_DIR)/signal -name \*.c))
 LIBC_BOTTOM_HALF_CRT_SOURCES = $(wildcard $(LIBC_BOTTOM_HALF_DIR)/crt/*.c)
-- 
2.42.0

